FirstProject/css/feedback.css

```diff
@@ -89,15 +78,16 @@ h1.interview_text_area {
     }
 
     .left_image {
-        float: left!important;
+        width: 100%;
     }
 
     header form {
         margin-left: 0!important;
     }
 
     .header_form_section {
-        width: 94%!important;
+        width: 87%;
+        margin: 0 40px auto auto;
```

Comment:
I typed the code like that and it worked for me. The code makes what I want :))

So.. :)) 
Is this technique wrong?